### PR TITLE
Watch later

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/components/cards/SingleCard.jsx`
`src/context/watchlater-context.js`
`src/functions/get-icon.js`
`src/functions/hooks/useDeleteFunction.js`
`src/page/watchlater/WatchLater.jsx`

## Thank you for reviewing the PR

### CODE REVIEW

File: src/backend/controllers/WatchLaterController.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  );
+};
+
+/**
+ * This handler handles removing videos from user's watchlater playlist.
+ * send DELETE Request at /api/user/watchlater/:videoId
+ * */
+
+export const removeItemFromWatchLaterVideos = function (schema, request) {
+  const user = requiresAuth.call(this, request);
```
Comment (VLeads) [inline src/backend/controllers/WatchLaterController.js:76]: It's good that you are writing comments too for understanding !!

**Review by VLeads — APPROVED** [review]
Code is easy to understand. Good to go !

File: src/backend/controllers/WatchLaterController.js (hunk 1 above)
Comment (Developers-codz) [reply src/backend/controllers/WatchLaterController.js:76]: This code is from backend section 😅

**Review by Maianki — APPROVED** [review]
Looks good to merge. Nothing much to suggest code is good and readable. Great job on UI too.

File: src/backend/controllers/WatchLaterController.js (hunk 1 above)
Comment (VLeads) [reply src/backend/controllers/WatchLaterController.js:76]: My bad 😅🤣

**Pull request merged**